src/elements/shared-styles.html

```diff
@@ -57,7 +57,7 @@
         /* Tags */
         --general: #9E9E9E;
         --android: #78c257;
-        --web: #2196f3;
+        --web: #1577C6;
```

Comment:
This web color doesn't _quite_ have enough contrast against the white background to satisfy color contrast needs:

* Before: https://webaim.org/resources/contrastchecker/?fcolor=2196F3&bcolor=FFFFFF
* After: https://webaim.org/resources/contrastchecker/?fcolor=2196F3&bcolor=FFFFFF